src/components/Filter.js

```diff
@@ -1,6 +1,21 @@
 import React from 'react'
 
 class Filter extends React.Component {
+  // What SpiceList passed down to Filter via props: 
+    // search={this.state.search} => this.props.search
+    // fourStarOnly={this.state.fourStarOnly} => this.props.fourStarOnly
+    // changeSearch={this.changeSearch} => this.props.changeSearch
+    // changeFourStar={this.changeFourStar} => this.props.changeFourStar
+
+  // Calls on changeSearch function passed down from SpiceList parent, passing in the current search value that user inputted
+  handleSearch = event => this.props.changeSearch(event.target.value);
+
+  // Calls on changeFourStar function passed down from SpiceList parent, passing in true if 4 Star Only box is checked and false otherwise
+  handleFourStar = event => {
+    event.target.checked
+      ? this.props.changeFourStar(true) 
+      : this.props.changeFourStar(false);
+  }
```

Comment:
You can shorten this up a bit, since the `checked` property will always give a boolean value there's no need to add the ternary:

```js
handleFourStar = event => this.props.changeFourStar(event.target.checked) 
```